team-norms/retrospectives.md

```diff
@@ -0,0 +1,52 @@
+# Retrospectives
+
+These retrospectives occur every 2 weeks on a Friday. There are 2 hours booked for this purpose. 
+
+There are two elements to retrospectives:
+
+* Retrospective (smaller groups performing retrospectives)
+* Comradrospective (whole company retrospective format)
+
+## Purpose 
+
+The purpose of retrospectives is to:
+
+* Provide a conduit to allow continuous improvement to be facilitated safely within teams.
+* Examine outcomes from previous retrospective, and understand what went well, and what didn’t.
+
+The purpose of the comradrospective is to:
+
+*  Examine previous action points, what went well and what didn’t.
```

Comment:
could we add here "Examine previous action points [from the retrospective]" to make it more obvious?